src/graphql/query/flssQuery.graphql

```diff
@@ -1,34 +1,21 @@
 #import '../fragments/loanCardFields.graphql'
 
-query basicLoanData(
-	$offset: Int,
-	$limit: Int,
-	$filters: LoanSearchFiltersInput,
+query flssLoanData(
 	$imgDefaultSize: String = "w480h360",
 	$imgRetinaSize: String = "w960h720",
-	$sortBy: LoanSearchSortByEnum,
-	$promoOnly: BasketInput,
 ) {
-	lend {
-		loans(
-			offset: $offset,
-			limit: $limit,
-			filters: $filters,
-			sortBy: $sortBy,
-			promoOnly: $promoOnly
-			) {
-			totalCount
-			values {
-				id
-				...loanCardFields
+  fundraisingLoans (filters: {sector: {eq: "retail"}} ) {
+    totalCount
+    values {
+        id
+		...loanCardFields
 				image {
 					id
 					default: url(customSize: $imgDefaultSize)
 					retina: url(customSize: $imgRetinaSize)
 					hash
 				}
-			}
-		}
-	}
+      }
+    }
 }
-
+s
```

Comment:
This tabbing on this file looks a little off and there is an `s` at the end here - is that on purpose?

Reply:
hmm, maybe that happened during conflict/merge resolution in github's ui. b/c @mcstover 's git commit linting hooks wouldn't let me get by with that! Will lint and push again.